packages/loopring_v3/contracts/impl/Exchange.sol

```diff
@@ -367,16 +367,65 @@ contract Exchange is IExchange, Claimable, ReentrancyGuard
     }
 
     function commitBlock(
-        uint8 blockType,
+        uint8  blockType,
         uint16 blockSize,
-        bytes calldata data,
+        uint8  blockVersion,
+        bytes calldata /*data*/,
         bytes calldata offchainData
         )
         external
         onlyOperator
         nonReentrant
     {
-        state.commitBlock(blockType, blockSize, data, offchainData);
+        // Decompress the data here so we can extract the data directly from calldata
+        bytes4 selector = IDecompressor(0x0).decompress.selector;
```

Comment:
Why 0x0?

Reply:
The address doesn't matter, it just needs to be something valid so the function selector can be calculated using solidity (it could also be calculated manually, but this is more clear). The function selector value is independent of the address of the contract.